src/features/dashboard/components/Dialogs/DeleteAppDialog/delete-app-dialog.scss

```diff
@@ -0,0 +1,44 @@
+.modal-wrap {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: rgba(0, 0, 0, 0.5);
+  z-index: 1000;
+  width: 512px;
+}
+
+.modal-icon {
+  display: flex;
+  justify-content: center;
+  padding: 24px 0px;
+  align-items: center;
+  background: var(--core-color-solid-red-100, #ffe6e6);
+}
+
+.content {
+  padding: 24px 32px 24px 32px;
+  text-align: center;
+  align-items: center;
+
+  h4 {
+    margin-bottom: 32px;
+    color: #333;
```

Comment:
Again, better to check in color palette if this color is available.
Adding color codes is not recommended... either find from palette or create a variable.

@agrim-deriv